lib/sidekiq/statistic/middleware.rb

```diff
@@ -30,18 +30,26 @@ def save_entry_for_worker(worker_status)
         time = worker_status[:last_runtime]
         realtime_hash = "#{REDIS_HASH}:realtime:#{time.sec}"
         worker_key = "#{time.strftime "%Y-%m-%d"}:#{status.delete :class}"
+        timeslist_key = "#{worker_key}:timeslist"
+        times_list_length = nil
```

Comment:
@davydovanton I tried a few options for a default value, but since `redis.pipelined` returns a [Future](https://github.com/redis/redis-rb/blob/master/lib/redis/pipeline.rb#L106) object I think I'd have to be doing type checking down below. Looking at the redis-rb code I believe any connection issues with redis would raise out of here making any default we set redundant. I think?